static/docs/commands-reference/remote:modify.md

```diff
@@ -176,7 +176,8 @@ For more information about the variables DVC supports, please visit
 
     1) `user` specified in one of the dvc configs;
     2) `user` specified in the url(e.g. `ssh://user@example.com/path`);
-    3) current user;
+    3) `user` specified in `~/.ssh/config` for remote host;
```

Comment:
minor question: is there a similar config on Windows? Should we specify a default path here as well?

Reply:
@shcheklein Good point. I thought that it would expand it to something like `C:\Users\user\.ssh\config`(which is correct), but apparently it is not able to handle slashes properly, so we also need to adjust that in the patch to use os.path.join.